scripts/sv/copy_sv_results.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# This script copies SV analysis results from a Google Dataproc cluster
+# to an appropriate bucket/directory on GCS. It also uploads contents of
+# local output logs
+
+# terminate script on error or if a command fails before piping to another command
+set -eu
+set -o pipefail
+
+if [[ "$#" -lt 3 ]]; then
+    echo -e
+"Please provide:
+  [1] GCS project name (required)
+  [2] GCS cluster name (required)
+  [3] cluster output directory (required)
+  [4] GCS user name (defaults to local user name)
+  [5] path to local log file (default to empty, i.e. no log)
+  [*] additional arguments that were passed to
+      StructuralVariationDiscoveryPipelineSpark"
+  exit 1
+fi
+
+PROJECT_NAME=$1
+CLUSTER_NAME=$2
+OUTPUT_DIR=$3
+GCS_USER=${4:-${USER}}
+LOCAL_LOG_FILE=${5:-"/dev/null"}
+
+shift 5
```

Comment:
Same concern: if we only provide 3 required arguments and the rest is java arguments, are 4 and 5 "shifted" away.